### ISSUE TITLE
What happened do dbandstra?

### ISSUE THREAD

**Issue by ghost** [catdevnull/awesome-zig#12]
He apparently created multiple nice Zig things but his account got deleted.

I actually played his Wasm game, "Oxid", in my browser and had lots of fun. Sad to see that he's gone. Maybe he changed his name or something?

**Comment by catdevnull** [comment]
I have no idea :( Have you asked in the IRC, Discord or some other part of the community?

**Comment by catdevnull** [comment]
OH NO!

**Comment by orhun** [comment]
I asked to same question recently. He created very cool Zig tools/libraries but then vanished from the surface of the internet. I came across this `zig-wav` library which was originally written by `dbandstra` but now the repo is deleted. I managed to get a copy from [here](https://github.com/marler8997/zig-wav) but the repository owner and the commits are totally different.

![Screenshot 2023-03-18 at 21-12-46 Commits · marler8997_zig-wav](https://user-images.githubusercontent.com/24392180/226140428-295dcbce-1e22-42d7-a5e0-676f96434821.png)

Initial commits goes back to 2019 but [`JeansSoft`](https://github.com/JeansSoft) account was created in 2022. This shows `dbandstra@protonmail.com` email might be linked to `JeansSoft` account somehow.

One month ago I reached out to the owner of the new `zig-wav` repository ([`marler8997`](https://github.com/marler8997)) via email but couldn't get a response yet.

In the end I still don't know what happened to `dbandstra`. I will probably search more clues on the internet as I come across more Zig stuff from him.
